Sming/SmingCore/Platform/OSTimer.h

```diff
@@ -0,0 +1,101 @@
+/****
+ * Sming Framework Project - Open Source framework for high efficiency native ESP8266 development.
```

Comment:
Offtopic: Please, don't mark review comments as resolved. Once you submit changes to PR the comments will collapse and I will know where to look more carefully and where to go briefly.